web/components/NavBarLoggedInDesktop.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { ReactElement } from "react";
+import { NavDefaults } from "@/display-content/NavDefaults";
+import { useUserData } from "@/lib/data-hooks/useUserData";
+import { Icon } from "@/components/njwds/Icon";
+import { AuthButton } from "@/components/AuthButton";
+import Link from "next/link";
+
+export const NavBarLoggedInDesktop = (): ReactElement => {
+  const { userData } = useUserData();
+
+  const getUser = (): string => {
+    return userData?.user.name
+      ? userData?.user.name
+      : userData?.user.email
+      ? userData?.user.email
+      : NavDefaults.userNotSetText;
+  };
+
+  return (
+    <div className="grid-container">
+      <div className="grid-row grid-gap">
+        <div className="desktop:grid-col-12 usa-prose">
+          <header className="flex fac fjb">
+            <div className="flex-custom">
+              <img className="padding-top-1" src="/img/Navigator-logo.svg" alt="Business.NJ.Gov Navigator" />
+            </div>
+            <div className="margin-left-auto flex fac">
+              <span className="white-space-no-wrap padding-right-1">
+                <span>
+                  <Link href={""}>
```

Comment:
Is this link supposed to navigate somewhere?

Reply:
The link component and button that is inside the link will be replaced with a drop down menu (#221). It's not suppose to go anywhere. I can remove it, comment it out, or leave as is in this story - do you have a preference?

Reply:
🤷 If it's not a link to anything, I'd suggest just remove the link component? It won't become a link when the dropdown happens, right, so it seems irrelevant? Maybe I'm misunderstanding? 

As a rule, I would definitely never want commented-out code on main 🙃

Reply:
I've removed the Link and updated the button to a div. In the future we may need to create button components for primary, secondary and tertiary buttons. The AuthButton for logout is using two different sets of styling (one set in the Hero component and another set in the Nav Bar).